ChangeLog.d/fix-pkwrite-buffer-overrun.txt

```diff
@@ -0,0 +1,9 @@
+Security
+   * Fix a buffer overflow in mbedtls_pk_write_pubkey(),
+     mbedtls_pk_write_pubkey_der() and mbedtls_pk_write_key_der().
```

Comment:
Not `pubkey`, per my revised understanding and testing. The affected functions are `mbedtls_pk_write_key_der()` and `mbedtls_pk_write_key_pem()`.

Reply:
As as noted by Janos, the PEM version is actually not vulnerable. So just `mbedtls_pk_write_key_der`.